id-repository/id-repository-core/src/test/resources/application-test.properties

```diff
@@ -363,6 +363,28 @@ logging.level.org.springframework=OFF
 logging.level.root=OFF
 spring.main.banner-mode=off
 
+mosip.idrepo.cache.names=credential_transaction,partner_extractor_formats,datashare_policies,topics,online_Verification_Partners,uin_encrypt_salt,uin_hash_salt,id_attributes
+#spring.cache.type=redis
+spring.cache.cache-names=${mosip.idrepo.cache.names}
+spring.cache.type=simple
+mosip.esignet.cache.size={'credential_transaction' : 200, \
+'partner_extractor_formats': 200, \
+'datashare_policies': 200, \
+'topics': 200, \
+'online_Verification_Partners': 200, \
+'uin_encrypt_salt' : 100, \
+'uin_hash_salt': 100 , \
+'id_attributes': 200 }
+
+# Cache expire in seconds is applicable for both 'simple' and 'Redis' cache type
+mosip.esignet.cache.expire-in-seconds={'credential_transaction' : 86400, \
```

Comment:
Where is this property used? can you replace esignet with idrepo?

Reply:
replaced it sir.